examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -0,0 +1,259 @@
+"""
+Functional connectivity kinds for group analysis of connectomes
+===============================================================
+
+This example compares different kinds of functional connectivity between
```

Comment:
You can keep it if you are comfortable with saying 'kind' but 'coefficients' might be more relevant I guess ?

For example, 'compares different measures/estimations of functional connectivity coefficients between .....' ?